DQM/Physics/src/EwkTauDQM.h

```diff
@@ -60,6 +60,7 @@ class EwkTauDQM : public DQMOneEDAnalyzer<> {
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"

```

Comment:
@Dr15Jones, do we want to `throw` here as we did in https://github.com/cms-sw/cmssw/pull/32847/files#diff-2401eb8162fcd6c8bbb94313bccfaa14e6e7e561717088d3a7c21c8f8ccd2ec0R158

Reply:
It looks like the author explicitly doesn't want to throw since they check the result of the 'getByLabel' and only if it failed do they print the message. Plus they have special code limiting the number of messages output.

Also, this is a module and therefore not under the same constraints as a DataFormat object which is included in many places.